tests/integration_tests/__init__.py

```diff
@@ -95,20 +118,6 @@ def tearDown(self):
         ):
             self.driver.execute_script("sauce:job-result=passed")
 
-        super(TestUI, self).tearDown()
-
-    def test_login_page(self):
-        """Ensure login page loads successfully"""
-
-        page = LoginPage(self.driver)
-        page.get("/user/sign-in")
-
-        assert "Uh-oh" not in page.w.find_element_by_tag_name("body").text
-
-    def test_login_form_fb_exists(self):
-        """Ensure Facebook button present on login form"""
-
-        page = LoginPage(self.driver)
-        page.get("/user/sign-in")
+        self.assertEqual([], self.verificationErrors)
```

Comment:
This seems to be exported from Selenium IDE, but I don't see any tests using it. I'm also hesitant about having an assertion outside of a test function.

Can you remove it?

Reply:
@ivan-c I think we should keep this so adding tests is really simple. If we remove this, Warren (and others) will have to worry about setup and teardown which makes things slightly more complicated. Having this will let him simply copy and paste just the test and avoid worrying about any other functions (I think). That was my goal for these changes.

Reply:
Sounds reasonable. Until some tests start referencing that property, I suggested a comment to make it obvious what it's there for.